sdks/jreleaser-tool-java-sdk/src/main/resources/META-INF/jreleaser/tools/jbang.properties

```diff
@@ -0,0 +1,29 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright 2020-2025 The JReleaser authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+unpack=true
+#should really use "latest"
```

Comment:
Would love to use latest but then version matching has to change

Reply:
having download be for specific version seems fine - but that it seem to want to download older version when newer available is problematic...anyhow - for now i suggest we just keep this and improve tool version handling separately.